### PR TITLE
Make harview uploadable to pypi

### PR DESCRIPTION
**Pull request by MichaelAquilina**
This should allow you to run

```
python setup.py sdist upload
```

which will upload it to pypi. This will then installable by anyone using `pip install harview`

I currently own the harview pypi name https://pypi.python.org/pypi/harview/0.1, but I only uploaded it to save it from being taken. I will delete this as soon as you are ready to upload it.

This PR also makes it Python3 compatible

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+setup(
+    name='harview',
+    version='0.2.0',
+    description=(
+        'A commandline tool which takes as input a .har (HTTP Archive) file '
+        'and dumps a human-readable summary of it to the console'
+    ),
+    long_description='',
+    license='GPLv3',
```
Comment (MichaelAquilina) [inline setup.py:19]: I'm not entirely sure what the License should be. I put this here, but let me know what you want me to update it to

Comment (fboender) [reply setup.py:19]: Hi Michael. Awesome work! The license should be MIT though. It's in the fine print all the way at the bottom of the README. I'll fix it my self and include a LICENSE file.

**Comment by MichaelAquilina** [comment]
Let me know when its ready to upload and I'll remove myself from package ownership :)

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    scripts=[
+        'src/harview',
+    ],
+    classifiers=[
+        'Environment :: Console',
+        'Intended Audience :: Developers',
+        'Natural Language :: English',
+        'License :: OSI Approved :: GNU General Public License v3 (GPLv3)',
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 3',
```
Comment (MichaelAquilina) [inline setup.py:31]: just realised you probably also want to update this to include a section for python 2 since its supported

**Comment by fboender** [comment]
> I'll remove myself from package ownership :)

Can you elaborate on how that works? You remove yourself from ownership, but how will PyPi know that I'm the new owner?

Regards

**Comment by MichaelAquilina** [comment]
This is the first time I have done this before but essentially all I need to do is delete the package from pypi account. That should free it up for you.

All you need to do is
1) create an account on pypi (https://pypi.python.org/pypi)
2) login to pypi by creating a `~/.pypirc` file with the following contents
```
[pypi]
repository=https://pypi.python.org/pypi
username=your_username
password=your_password
```
3) Run `python setup.py sdist upload` on this project

When you are ready to run 3 just let me know

**Comment by MichaelAquilina** [comment]
Any success with this @fboender ?

**Comment by fboender** [comment]
Sorry, been super busy! I'm ready to reregister the package and upload it as soon as you've deleted your version.

**Comment by MichaelAquilina** [comment]
Done! Go for it :)

**Comment by fboender** [comment]
Had to bump the version number, but other than that, it was smooth sailing:

    running upload
    Submitting dist/harview-0.2.2.tar.gz to https://upload.pypi.org/legacy/
    Server response (200): OK

Thanks for the help!

**Comment by MichaelAquilina** [comment]
Glad to help and great to hear! :)

**Pull request merged**